react/package-lock.json

```diff
@@ -1,1759 +1,1031 @@
 {
     "name": "hazmapper",
     "version": "0.0.0",
-    "lockfileVersion": 3,
```

Comment:
can you recreate your package-lock.json.  you should switch to a newer version of npm (via nvm). like npm version 20.x

Reply:
Sure